docs/eigencompute/howto/operate/terminate-application.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Terminate Application
+sidebar_position: 5
+---
+
+:::danger Irreversible Action
+Termination is permanent and irreversible. The TEE wallet mnemonic becomes inaccessible. Any funds in the wallet will be lost forever.
+:::
+
+## Terminate an app (with confirmation)
+
+```bash
+eigenx app terminate my-app
+```
+
+## Terminate without confirmation
+
+```bash
+eigenx app terminate --force my-app
+```
+
+## Terminate by ID
+
+```bash
+eigenx app terminate app-1a2b3c4d5e6f
+```
+
+### When to Use
+
+#### Clean up test apps
+
+```bash
+# Remove test deployment
+eigenx app terminate test-deployment
```

Comment:
I don't understand why we have to do write this like a cleanup test app. It doesn't make sense because we already have terminate commands on top.